libs/design/docs/theming.md

```diff
@@ -0,0 +1,36 @@
+# Theming
+
+The **daff-theme** utility library is for theming the daffodil
+design library. It contains many useful mixins, functions, 
+and variables.
+
+## Step 1: Add the path to your angular configuration file.
+Within your project's `angular.json`, append the following to the key `projects.project-name.architect.build.options`
```

Comment:
should be 
"...to the `projects.project-name.architect.build.options` key:" 
or 
"...to the key, `projects.project-name.architect.build.options`:"